decisions/wc_order_decisions.txt

```diff
@@ -250,8 +250,54 @@ decisions = {
 			factor = 1
 		}
 	}
-}
 	
+	create_blood_knights_order = {
+		ai_check_interval = 12
+		is_high_prio = yes
+		
+		only_playable = yes
+		
+		potential = {
+			AND = { religion = cult_of_sunwell
+				
+			    OR = {
+				    has_landed_title = e_quelthalas
+				    has_landed_title = k_quelthalas
+				}
+				OR = {
+				    culture = high_elf
+					culture = blood_elf
+				}
+				OR = {
+					trait = creature_high_elf
+					trait = creature_blood_elf
+				}
+			}
+			d_blood_knights_order = { has_holder = no }
+		}
+		effect = {
+			narrative_event = { id = WCBKO.01 }
+		}
+		allow = {
+		    trigger_if = {
+                limit = { has_global_flag = blood_elves_happened_flag }
+				num_of_holy_sites = 3
+				piety = 250
+			}
+            trigger_else = { 
+				num_of_holy_sites = 4
+				piety = 2000
```

Comment:
That's A LOT of piety. Even reforming religion requires 750.

Reply:
If anything, forming a holy order for eastern religions in vanilla require 1000.

Reply:
Changed back to 1000